src/auth_manager/security_config.fbs

```diff
@@ -29,13 +32,18 @@ table AuthManager {
     tf_token_url: string;
 
     // leeway to the token expiration
-    leeway: uint32 = 0;
+    expiry_leeway_secs: uint32 = 0;
+
+    // leeway to the token issued_at
+    iat_leeway_secs: uint32 = 5;
+
+    // leeway to the token not_before
+    nbf_leeway_secs: uint32 = 5;
```

Comment:
Should not nbf be 0?

https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.5

Reply:
NVM